---
challenge-title: EcoTox TARGET Challenge
layout: front-matter-data
permalink: /challenge/ecotox-challenge/
challenge-id: 1088
status: open
sidenav: true
card-image: /assets/images/cards/Challenge_TARGET_Logo.png
agency-logo: 256px-Environmental_Protection_Agency_logo.png
tagline: Develop high quality, low-cost tools that assess global gene expression
  in common aquatic toxicity test organisms
agency: Environmental Protection Agency
partner-agencies-federal: U.S. Army Engineer Research and Development Center
partners-non-federal: DoW, Environment and Climate Change Canada, European
  Commission Joint Research Centre, Syngenta
external-url: null
total-prize-offered-cash: $300,000
type-of-challenge: Scientific
submission-start: 2020/03/19 12 p.m.
submission-end: 2021/12/03 11:59 p.m.
fiscal-year: FY20, FY21
legal-authority: Section 20 of the Federal Insecticide, Fungicide, and
  Rodenticide Act, 7 U.S.C. 136r; Section 10 of the Toxic Substances Control
  Act, 15 U.S.C. 2609; as well as Section 104 of the Clean Water Act, 33 U.S.C.
  1254
challenge-manager: Denice Shaw
challenge-manager-email: shaw.denice@epa.gov
point-of-contact: villeneuve.dan@epa.gov
description: <p>This challenge calls for respondents to develop high quality,
  low cost, technologies/platforms for evaluating global gene expression in
  samples from four common aquatic toxicity test organisms:&nbsp; Pimephales
  promelas (a fish), Daphnia magna (a crustacean), Chironomous dilutus (an
  insect; formerly Chironomous tentans), and Raphidocelis subcapitata (a green
  algae). These represent species and associated trophic levels most frequently
  tested when evaluating the ecological hazards of chemicals. While there are
  many extant, viable, technologies for measuring global gene expression, the
  significant technological leap and challenge to the solver community is to
  provide these capabilities at a cost and scale of commercial throughput that
  can accommodate the analysis of thousands or tens of thousands of samples per
  year. A target price point is $50 per sample or less.</p> <p><strong>Key
  Dates:</strong></p> <ul> <li><a
  href="https://www.epa.gov/innovation/ecotox-target-challenge-informational-webinar"
  target="_blank" rel="noopener">Informational Webinar</a>:&nbsp; 12:00 p.m. ET,
  February 18, 2020</li> <li>Registration Closes:&nbsp; 11:59 p.m. ET, March 16,
  2020</li> <li>Kick-off Webinar for Registered Solvers:&nbsp; 12:00 p.m. ET,
  March 19, 2020</li> <li>Submission Start:&nbsp; 12:00 p.m. ET, March 19,
  2020</li> <li>Submission End:&nbsp; 11:59 p.m. ET June 15, 2021</li> </ul>
prizes: <p>The prize for the challenge winner will be $300,000 US dollars. One
  prize will be awarded.</p>
rules: <p>To be eligible to compete for the award, prospective solvers must
  register by no later than March 16, 2020 at:&nbsp; <a
  href="https://www.epa.gov/innovation/ecotox-target-challenge" target="_blank"
  rel="noopener">https://www.epa.gov/innovation/ecotox-target-challenge</a>.</p>
  <h4>Eligibility:</h4> <ul> <li>Eligible:&nbsp; Individuals, or teams from
  private companies, academic institutions, non-governmental organizations, or
  independent research or technological institutes. The competition is open to
  both U.S. and foreign citizens/organizations.</li> <li>Not eligible:&nbsp;
  U.S. or foreign government organizations.</li> <li>Not eligible:&nbsp;
  Individuals involved in development of award selection criteria or reference
  sample generation.</li> </ul> <p>For additional details and rules,
  visit:&nbsp; <a href="https://www.epa.gov/innovation/ecotox-target-challenge"
  target="_blank"
  rel="noopener">https://www.epa.gov/innovation/ecotox-target-challenge</a>.</p>
  <h4>Terms and Conditions</h4> <p>By registering, Solvers agree to the
  following terms of participation:</p> <ul> <li>Solvers will not receive
  compensation for resources or time invested in addressing the challenge. Only
  the winning Solution will receive a cash award.</li> <li>Only the top-ranked
  Solution will receive the award.</li> <li>Solvers retain their rights to all
  intellectual property (e.g., details and design of their technology) that may
  be disclosed to the sponsors over the course of the challenge. Technical
  details and designs will not be disclosed or published without permission from
  the technical point of contact named in the registration.</li> <li>Sponsors
  retain the right to disclose reference sample data, performance criteria, and
  other evaluation criteria summarized in the technology description template to
  provide a transparent reporting of how the winning solution was selected.</li>
  <li>Sponsors retain the right to publish, present, and/or otherwise publicize
  results of the challenge competition that does not involve the disclosure of
  intellectual property of the Solver(s). Solvers will be afforded opportunity
  to review publications, presentation, or other publicity to protect against
  unwanted disclosure of intellectual property.</li> <li>Solvers reserve the
  right to remove themselves from the competition at any time, up to final
  submission of results for evaluation, by notifying the sponsor in writing. The
  technical point of contact must make the request in writing on behalf of
  his/her team. </li> <li>Registration for the challenge does not confer any
  obligation to deliver results.  However, any solvers removing themselves from
  the competition prior to evaluation forfeit the rights to publish results
  obtained for the reference samples supplied for the competition unless they
  obtain written consent from the challenge sponsors.</li> <li>Solvers that do
  not submit their results and technology description template by the submission
  deadline will be automatically removed from the competition and subject to the
  same terms as if they had forfeited in writing. The submission deadline may be
  extended at the discretion of the sponsors, but any extension will apply to
  all registered solvers.</li> </ul>
judging: <h4>Judging Panel</h4> <p>The judging panel will consist of six subject
  matter experts selected by the Challenge Sponsors.</p> <h4>Judging
  Criteria</h4> <p>Submissions will be judged based on data generated for a
  common set of blinded reference samples provided to all Solvers and an
  accompanying Technology Description Template.</p> <h4>Scoring Overview</h4>
  <p>Scoring will be based on the weighted (% of total score) criteria provided
  in the tables below. Each criterion is scored based on either a nominal or
  fractional scale of 0-5 with 0 being the lowest and 5 being the highest, or by
  pass/fail with 5 being pass and 0 being fail.</p> <table width="99%"
  class="usa-table"> <tbody> <tr> <td width="72%"> <p><strong>1. Quality and
  performance</strong></p> </td> <td width="15%"> <p>Points</p> </td> <td
  width="12%"> <p>Weighting</p> </td> </tr> <tr> <td width="72%">
  <p><strong>Quality control:&nbsp; </strong>Does the platform contain a quality
  control system that addresses consistency within and between samples and is
  consistent with current standards used within various platforms for
  transcriptomic analyses?</p> <p>o E.g., Microarray chip-based platforms should
  contain hybridization controls, redundant positional controls to evaluate edge
  effects, etc.</p> <p>o E.g., RNA-seq platforms provide number of reads per
  sample, base quality score by cycle, nucleotide distribution by cycle, GC
  content, etc.</p> <p>o Note &ndash; Scoring for this category will require
  subjectivity from the judging panel.</p> </td> <td width="15%"> <p>0 to 5</p>
  </td> <td width="12%"> <p>10%</p> </td> </tr> <tr> <td width="72%">
  <p><strong>Data collection/extraction:&nbsp; </strong>Are the data
  collection/extraction methods and expression normalization/quantification
  methods described in adequate detail? Are they compatible with the ToxCast
  high throughput transcriptomics data analysis pipeline?</p> </td> <td
  width="15%"> <p>0 to 5</p> </td> <td width="12%"> <p>5%</p> </td> </tr> <tr>
  <td width="72%"> <p><strong>Precision:&nbsp; </strong>Precision will be
  determined by evaluating 1) coefficients of variation of gene expression
  values across unblinded technical duplicates, 2) correlation analysis of
  fold-change profiles between selected reference samples, resulting in a metric
  of concordance, and 3) clustering unblinded reference samples when analyzed
  together with all conditions. Results from these three analyses will be
  normalized and merged into a multiplier between 0-1 that will be used to
  determine the total score between 0-5.</p> </td> <td width="15%">
  <p>multiplier x 5</p> </td> <td width="12%"> <p>10%</p> </td> </tr> <tr> <td
  width="72%"> <p><strong>Accuracy:&nbsp; </strong>Accuracy will be determined
  by evaluating 1) the percent concordance between fold-change values determined
  by Solver data compared to the fold-change values determined by
  pre-qualification. Results to determine the total score between 0-5.</p> </td>
  <td width="15%"> <p>0 to 5</p> </td> <td width="12%"> <p>10%</p> </td> </tr>
  <tr> <td width="72%"> <p><strong>Quantity of RNA:&nbsp; </strong>Was the
  quantity of reference RNA used per analysis tracked and reported (Y = points
  awarded; N=0 points)?</p> <p>o Results generated using &lt; 1 &micro;g total
  RNA = 2 pts</p> <p>o Results generated using &lt; 0.25 &micro;g total RNA = 1
  pt</p> <p>o Results generated using &lt; 0.1 &micro;g total RNA = 1 pt</p>
  <p>o Results generated using &lt; 0.01 &micro;g total RNA = 1 pt</p> </td> <td
  width="15%"> <p>&nbsp;0 to 5</p> </td> <td width="12%"> <p>5%</p> </td> </tr>
  </tbody> </table> <hr> <table width="99%" class="usa-table"> <tbody> <tr> <td
  width="72%"> <p><strong>2. Economic and commercial viability</strong></p>
  </td> <td width="15%"> <p>Points</p> </td> <td width="12%"> <p>Weighting</p>
  </td> </tr> <tr> <td width="72%"> <p><strong>Economic viability </strong>
  (i.e., cost per sample, including downstream data analysis cost):&nbsp; </p>
  <p>o Is the cost of sample preparation and per sample supply and reagent costs
  for conducting the sample analysis and generating the data provided? If
  proprietary downstream data analysis software is required, include a
  per-sample adjustment to overall sample cost based on software license
  cost.</p> <p>o Total per-sample cost is</p> <ul> <li>i. $20 or less = 5
  pts</li> <li>ii. &gt;$20-$30 = 4 pts.</li> <li>iii. &gt;$30-$50 = 3 pts.</li>
  <li>iv. &gt;$50-$75 = 2 pts.</li> <li>v. &gt;$75-$100 = 1 pt.</li> <li>vi.
  &gt;$100 = 0 pts.</li> </ul> </td> <td width="15%"> <p>0 to 5</p> </td> <td
  width="12%"> <p>20%</p> </td> </tr> <tr> <td width="72%">
  <p><strong>Commercial viability and throughput capability:&nbsp; </strong>Is
  there a reasonable demonstration/description of how and when the Solver would
  be able to meet the potential throughput requirements of HTP sample
  generation?</p> </td> <td width="15%"> <p>0 to 5</p> </td> <td width="12%">
  <p>10%</p> </td> </tr> </tbody> </table> <hr> <table width="99%"
  class="usa-table"> <tbody> <tr> <td width="72%"> <p><strong>3.
  Coverage</strong></p> </td> <td width="15%"> <p>Points</p> </td> <td
  width="12%"> <p>Weighting</p> </td> </tr> <tr> <td width="72%">
  <p><strong>Approach and annotation:&nbsp; </strong></p> <p>o Is the approach
  taken for detection and quantification of transcript expression adequately
  described? (e.g., whether the platform employs a targeted or non-targeted
  analysis and the general means by which the platform detects and quantifies
  transcript presence and abundance)?</p> <p>o Are annotation files provided
  with each platform/species that contain the required information and link to
  the data files?</p> </td> <td width="15%"> <p>0 to 5</p> </td> <td
  width="12%"> <p>10%</p> </td> </tr> <tr> <td width="72%">
  <p><strong>Transcriptome coverage:&nbsp; </strong>What proportion of
  transcriptome coverage do the platforms have in relation to the
  pre-qualification standards? The mean percent coverage will be calculated
  across the four species&rsquo; platforms and used as a multiplier to determine
  point value.</p> </td> <td width="15%"> <p>multiplier x 5</p> </td> <td
  width="12%"> <p>20%</p> </td> </tr> <tr> <td width="72%"> <p><strong>Species
  coverage:&nbsp; </strong>Did the solvers provide a platform and reference
  sample data for all four species?</p> </td> <td width="15%"> <p>Y or N</p>
  </td> <td width="12%"> <p>*100%</p> </td> </tr> </tbody> </table> <p>*Eligible
  submissions will include platforms, data and associated required information
  for all four species.</p>
how-to-enter: <p>To be eligible to compete for the award, prospective solvers
  must register at <a
  href="https://www.epa.gov/innovation/forms/ecotox-target-challenge-registration"
  target="_blank"
  rel="noopener">https://www.epa.gov/innovation/forms/ecotox-target-challenge-registration</a>
  by no later than March 16, 2020.</p> <p>Submission:&nbsp; Instructions on
  submission of entries will be provided to registered participants during the
  Kick-off Webinar on March 19, 2020.</p>
prize: true
submission-link: http://www.epa.gov/innovation/EcoTox-Target-Challenge
---
<!-- Description start -->

### Description

<p>This challenge calls for respondents to develop high quality, low cost, technologies/platforms for evaluating global gene expression in samples from four common aquatic toxicity test organisms: Pimephales promelas (a fish), Daphnia magna (a crustacean), Chironomous dilutus (an insect; formerly Chironomous tentans), and Raphidocelis subcapitata (a green algae). These represent species and associated trophic levels most frequently tested when evaluating the ecological hazards of chemicals. While there are many extant, viable, technologies for measuring global gene expression, the significant technological leap and challenge to the solver community is to provide these capabilities at a cost and scale of commercial throughput that can accommodate the analysis of thousands or tens of thousands of samples per year. A target price point is $50 per sample or less.</p>
 <p><strong>Key Dates:</strong></p>
 <ul>
   <li><a href="https://www.epa.gov/innovation/ecotox-target-challenge-informational-webinar" target="_blank" rel="noopener">Informational Webinar</a>: 12:00 p.m. ET, February 18, 2020</li>
      <li>Registration Closes: 11:59 p.m. ET, March 16, 2020</li>
   <li>Kick-off Webinar for Registered Solvers: 12:00 p.m. ET, March 19, 2020</li>
   <li>Submission Start: 12:00 p.m. ET, March 19, 2020</li>
  <li>Submission End: 11:59 p.m. ET, December 3, 2021</li>
  </ul>

<!-- Prizes start -->

### Prizes

<p>The prize for the challenge winner will be $300,000 US dollars. One prize will be awarded.</p>

<!-- Rules start -->

### Rules

<p>To be eligible to compete for the award, prospective solvers must register by no later than March 16, 2020 at: <a href="https://www.epa.gov/innovation/ecotox-target-challenge" target="_blank" rel="noopener">https://www.epa.gov/innovation/ecotox-target-challenge</a>.</p>
<h4>Eligibility:</h4>
<ul>
  <li>Eligible: Individuals, or teams from private companies, academic institutions, non-governmental organizations, or independent research or technological institutes. The competition is open to both U.S. and foreign citizens/organizations.</li>
 <li>Not eligible: U.S. or foreign government organizations.</li>
 <li>Not eligible: Individuals involved in development of award selection criteria or reference sample generation.</li>
  </ul>
<p>For additional details and rules, visit: <a href="https://www.epa.gov/innovation/ecotox-target-challenge" target="_blank" rel="noopener">https://www.epa.gov/innovation/ecotox-target-challenge</a>.</p>
<h4>Terms and Conditions</h4>
<p>By registering, Solvers agree to the following terms of participation:</p>
<ul>
  <li>Solvers will not receive compensation for resources or time invested in addressing the challenge. Only the winning Solution will receive a cash award.</li>
<li>Only the top-ranked Solution will receive the award.</li>
<li>Solvers retain their rights to all intellectual property (e.g., details and design of their technology) that may be disclosed to the sponsors over the course of the challenge. Technical details and designs will not be disclosed or published without permission from the technical point of contact named in the registration.</li>
<li>Sponsors retain the right to disclose reference sample data, performance criteria, and other evaluation criteria summarized in the technology description template to provide a transparent reporting of how the winning solution was selected.</li>
<li>Sponsors retain the right to publish, present, and/or otherwise publicize results of the challenge competition that does not involve the disclosure of intellectual property of the Solver(s). Solvers will be afforded opportunity to review publications, presentation, or other publicity to protect against unwanted disclosure of intellectual property.</li>
<li>Solvers reserve the right to remove themselves from the competition at any time, up to final submission of results for evaluation, by notifying the sponsor in writing. The technical point of contact must make the request in writing on behalf of his/her team. </li>
<li>Registration for the challenge does not confer any obligation to deliver results.  However, any solvers removing themselves from the competition prior to evaluation forfeit the rights to publish results obtained for the reference samples supplied for the competition unless they obtain written consent from the challenge sponsors.</li>
<li>Solvers that do not submit their results and technology description template by the submission deadline will be automatically removed from the competition and subject to the same terms as if they had forfeited in writing. The submission deadline may be extended at the discretion of the sponsors, but any extension will apply to all registered solvers.</li>
  </ul>

<!-- Judging start -->

### Judging Criteria

<h4>Judging Panel</h4>
<p>The judging panel will consist of six subject matter experts selected by the Challenge Sponsors.</p>
<h4>Judging Criteria</h4>
<p>Submissions will be judged based on data generated for a common set of blinded reference samples provided to all Solvers and an accompanying Technology Description Template.</p>
<h4>Scoring Overview</h4>
<p>Scoring will be based on the weighted (% of total score) criteria provided in the tables below. Each criterion is scored based on either a nominal or fractional scale of 0-5 with 0 being the lowest and 5 being the highest, or by pass/fail with 5 being pass and 0 being fail.</p>
<table width="99%" class="usa-table">
<tbody>
<tr>
<td width="72%">
<p><strong>1. Quality and performance</strong></p>
</td>
<td width="15%">
<p>Points</p>
</td>
<td width="12%">
<p>Weighting</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Quality control: </strong>Does the platform contain a quality control system that addresses consistency within and between samples and is consistent with current standards used within various platforms for transcriptomic analyses?</p>
<p>o E.g., Microarray chip-based platforms should contain hybridization controls, redundant positional controls to evaluate edge effects, etc.</p>
<p>o E.g., RNA-seq platforms provide number of reads per sample, base quality score by cycle, nucleotide distribution by cycle, GC content, etc.</p>
<p>o Note &ndash; Scoring for this category will require subjectivity from the judging panel.</p>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>10%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Data collection/extraction: </strong>Are the data collection/extraction methods and expression normalization/quantification methods described in adequate detail? Are they compatible with the ToxCast high throughput transcriptomics data analysis pipeline?</p>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>5%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Precision: </strong>Precision will be determined by evaluating 1) coefficients of variation of gene expression values across unblinded technical duplicates, 2) correlation analysis of fold-change profiles between selected reference samples, resulting in a metric of concordance, and 3) clustering unblinded reference samples when analyzed together with all conditions. Results from these three analyses will be normalized and merged into a multiplier between 0-1 that will be used to determine the total score between 0-5.</p>
</td>
<td width="15%">
<p>multiplier x 5</p>
</td>
<td width="12%">
<p>10%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Accuracy: </strong>Accuracy will be determined by evaluating 1) the percent concordance between fold-change values determined by Solver data compared to the fold-change values determined by pre-qualification. Results to determine the total score between 0-5.</p>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>10%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Quantity of RNA: </strong>Was the quantity of reference RNA used per analysis tracked and reported (Y = points awarded; N=0 points)?</p>
<p>o Results generated using &lt; 1 &micro;g total RNA = 2 pts</p>
<p>o Results generated using &lt; 0.25 &micro;g total RNA = 1 pt</p>
<p>o Results generated using &lt; 0.1 &micro;g total RNA = 1 pt</p>
<p>o Results generated using &lt; 0.01 &micro;g total RNA = 1 pt</p>
</td>
<td width="15%">
<p>&nbsp;0 to 5</p>
</td>
<td width="12%">
<p>5%</p>
</td>
</tr>
</tbody>
</table>
<hr>
<table width="99%" class="usa-table">
<tbody>
<tr>
<td width="72%">
<p><strong>2. Economic and commercial viability</strong></p>
</td>
<td width="15%">
<p>Points</p>
</td>
<td width="12%">
<p>Weighting</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Economic viability </strong> (i.e., cost per sample, including downstream data analysis cost): </p>
<p>o Is the cost of sample preparation and per sample supply and reagent costs for conducting the sample analysis and generating the data provided? If proprietary downstream data analysis software is required, include a per-sample adjustment to overall sample cost based on software license cost.</p>
<p>o Total per-sample cost is</p>
<ul>
  <li>i. $20 or less = 5 pts</li>
  <li>ii. &gt;$20-$30 = 4 pts.</li>
  <li>iii. &gt;$30-$50 = 3 pts.</li>
  <li>iv. &gt;$50-$75 = 2 pts.</li>
  <li>v. &gt;$75-$100 = 1 pt.</li>
  <li>vi. &gt;$100 = 0 pts.</li>
  </ul>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>20%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Commercial viability and throughput capability: </strong>Is there a reasonable demonstration/description of how and when the Solver would be able to meet the potential throughput requirements of HTP sample generation?</p>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>10%</p>
</td>
</tr>
</tbody>
</table>
<hr>
<table width="99%" class="usa-table">
<tbody>
<tr>
<td width="72%">
<p><strong>3. Coverage</strong></p>
</td>
<td width="15%">
<p>Points</p>
</td>
<td width="12%">
<p>Weighting</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Approach and annotation: </strong></p>
<p>o Is the approach taken for detection and quantification of transcript expression adequately described? (e.g., whether the platform employs a targeted or non-targeted analysis and the general means by which the platform detects and quantifies transcript presence and abundance)?</p>
<p>o Are annotation files provided with each platform/species that contain the required information and link to the data files?</p>
</td>
<td width="15%">
<p>0 to 5</p>
</td>
<td width="12%">
<p>10%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Transcriptome coverage: </strong>What proportion of transcriptome coverage do the platforms have in relation to the pre-qualification standards? The mean percent coverage will be calculated across the four species&rsquo; platforms and used as a multiplier to determine point value.</p>
</td>
<td width="15%">
<p>multiplier x 5</p>
</td>
<td width="12%">
<p>20%</p>
</td>
</tr>
<tr>
<td width="72%">
<p><strong>Species coverage: </strong>Did the solvers provide a platform and reference sample data for all four species?</p>
</td>
<td width="15%">
<p>Y or N</p>
</td>
<td width="12%">
<p>*100%</p>
</td>
</tr>
</tbody>
</table>
<p>*Eligible submissions will include platforms, data and associated required information for all four species.</p>

<!--  How To Enter start -->

### How To Enter

<p>To be eligible to compete for the award, prospective solvers must register at <a href="https://www.epa.gov/innovation/forms/ecotox-target-challenge-registration" target="_blank" rel="noopener">https://www.epa.gov/innovation/forms/ecotox-target-challenge-registration</a> by no later than March 16, 2020.</p>
<p>Submission: Instructions on submission of entries will be provided to registered participants during the Kick-off Webinar on March 19, 2020.</p>