---
permalink: /toolkit/case-studies/cdc-no-petri-dish/
layout: toolkit
title: Case Study - No Petri Dish
---



<!--// OPEN #page-wrap //-->
<div id="page-wrap">


<div class="inner-page-wrap has-no-sidebar portfolio-type-standard row clearfix">

<!-- OPEN article -->
<article
class="portfolio-article col-sm-12 clearfix post-8207 portfolio type-portfolio status-publish has-post-thumbnail hentry portfolio-category-scientific portfolio-category-2-1 portfolio-category-2-6 portfolio-category-5-5"
id="8207" itemscope="" itemtype="http://schema.org/CreativeWork">



<div class="portfolio-item-content">


<div class="container port-detail-media-container"><!-- OPEN .container -->

<figure class="media-wrap col-sm-12">
</figure>

</div><!-- CLOSE .container -->

<div class="grid-container padding-bottom-5">

<section class="article-body-wrap col-sm-9">
<section class="portfolio-detail-description">
<div class="body-text clearfix" itemprop="description">

<section class="container">
<div class="row">
<div class="spb_content_element col-sm-12 spb_text_column">
<div class="spb_wrapper clearfix">
<h1>No Petri Dish</h1>
<h3 style="border-bottom: 1px solid #e4e4e4;"  class="spb-heading spb-text-heading"><span>Centers for Disease Control and Prevention (CDC)</span>
</h3>

<h2>Summary</h2>
<p>Surveillance of infectious diseases is an integral part of
public health prevention and control programs.&nbsp; A
number of these programs have successfully implemented
networks to detect and investigate outbreaks and to
implement effective, targeted controls.&nbsp; Examples of
these networks include PulseNet, the National Tuberculosis
Surveillance System and the Active Bacterial Core
Surveillance program, among others. Each of these programs
relies on the culture and testing of patient isolates at
community and clinical laboratories, and the subsequent
submission of these cultured isolates to state and local
public health laboratories for detailed analysis.</p>
<p>A new generation of commercial diagnostic tests are now
entering the marketplace that offer physicians faster
results, more types of information than were previously
available in some cases, and that can test for broad groups
of pathogens based on patient symptoms and presentation.&nbsp;
Unfortunately, unlike culture-based methods, these new tests
do not result in isolates being available for public health
purposes, and it is expected that this lack of isolates will
make it increasingly difficult or impossible to detect and
investigate outbreaks or other important public health
trends.&nbsp; In order to maintain and improve important
public health surveillance activities, new laboratory
methods are needed that do not depend on isolates or
cultures.</p>
<div id="attachment_2831" style="max-width: 460px"
class="wp-caption alignleft"><a
href="{{ site.baseurl }}/assets/images/toolkit/case-studies/no-petri-dish-case-study-e1474487034657.jpg"><img
class="wp-image-2831"
src="{{ site.baseurl }}/assets/images/toolkit/case-studies/no-petri-dish-case-study-e1474487034657.jpg"
alt="A pipette hovers over a petri dish."
sizes="(max-width: 450px) 100vw, 450px" width="450"
height="270"></a>
<p class="wp-caption-text">The "No Petri Dish" Challenge
asked an open-ended question and was broken into two
phases to evaluate submissions. (File Photo)</p></div>
<p>The Shiga-toxin-producing <em>Escherichia coli</em> (STEC)
represents a difficult challenge.&nbsp; STEC is similar in
most respects to commensal <em>E. coli</em> (i.e., strains
of <em>E. coli</em> normally found in the human gut).&nbsp;
Tracking the pathogen therefore requires reliably
identifying a variable marker that can be used for subtyping
and to link these data to other markers that can be used for
identification and characterization of the pathogen, all
within a "dirty" and complex sample matrix (i.e., a stool
sample).&nbsp; The challenge was to "describe a novel or
innovative method to strain-type and characterize pathogenic
organisms, such as STEC, directly from a complex clinical
sample, without the need for culture or culture-based
amplification."</p>
<p>Contestants submitted their solutions between Sept. 1, 2014
and Nov. 30, 2014. The winner was notified on Dec. 15, 2014.
An award of $200,000 went to the winning entrant. Only one
winner was selected.</p>
<p>This challenge asked an open-ended question, and thus, the
challenge organizers expected a wide range of proposed
solutions. The competition was consequently split into two
phases. In the first phase, all qualified submissions that
were received during the challenge period were submitted to
an expert review by a panel of CDC and external subject
matter experts.&nbsp; In the second phase, the leading
proposal from the technical review was invited to
participate in a timed challenge, using a blinded set of
synthetic datasets that had been designed by the technical
review panel. This two-phased approach allowed the No Petri
Dish Challenge to consider a wide range of potential
technical solutions, while enabling the reviewers to tailor
validation and verification of performance claims using
expert-curated data.</p>
<h2>Results</h2>
<p>The "No Petri Dish" Diagnostic Test Challenge attracted
considerable interest from both academia and industry
through social media and email inquiries, and received some
creative submissions and feedback during the contest's open
period. Eight submissions were received from seven project
teams.&nbsp; Two submissions were ineligible for the
challenge due to citizenship restrictions under the America
COMPETES Act. &nbsp;The remaining six submissions from
academic centers and nonprofit organizations presented a
wide range of solutions.</p>
<p>The technical review panel selected the solution that was
submitted by OneCodex, a YCombinator-backed startup that had
developed an elegant, cloud-based metagenomic analytical
platform.&nbsp; OneCodex already had been developing their
software platform at the time of the challenge, and they
were able to incorporate many new and useful features as a
result of the challenge requirements and subsequent CDC
feedback.&nbsp; The OneCodex team was then contacted and
invited to participate in a second and final challenge phase
before a final award determination was made.&nbsp; In this
second phase, CDC subject matter experts generated five
synthetic datasets that approximated whole-sample
metagenomic data&nbsp; from human stool with differing
levels of different reference strains of STEC.&nbsp; The
OneCodex team was given 24 hours to analyze the samples on
its platform, determine what strains (if any) were present
in each sample and provide whatever additional details and
characteristics (e.g., antimicrobial resistance genotype) it
could extract from the samples. While the OneCodex team did
not submit a perfect solution for all samples (all samples
were challenging; several were deliberately impossible), its
solution impressed the review committee, as did the quality
and clarity of the report they generated.</p>
<p>The OneCodex team was awarded the grand prize of $200,000 for
this challenge on Jan. 15, 2015.</p>
<h2>Areas of Excellence</h2>
<p><strong>Area of Excellence #1: "Design the
Challenge"</strong></p>
<p>This challenge posed a very open-ended and difficult
question, and the CDC challenge team recognized from the
outset that a wide range of potential solutions could be
proposed.&nbsp; As a result, the challenge was carefully
designed to include two phases: an initial technical and
scientific review by an expert panel and a
proof-of-viability phase, which would have been adjusted
according to the finalists from the technical review.&nbsp;
In the case of the No Petri Dish challenge, the finalist
proposed a bioinformatic analysis solution, and as such, the
technical review panel developed a blinded set of simulated
data from samples to validate the finalist's solution and
verify its claims of performance.&nbsp; Had one or more
laboratory-based solutions been proposed, the review panel
was also prepared to distribute a similarly blinded set of
simulated human clinical specimens.&nbsp; Upon submission of
a successful result to the timed Phase 2 of the challenge, a
single grand prize award was made to the winning team.</p>
<p>In retrospect, this design and incentive structure was ideal
for the challenge. The No Petri Dish Challenge was
essentially a single award, winner-takes-all challenge with
two phases.&nbsp; Because we asked a very open-ended
question and expected a wide range of proposed solutions, we
knew we would need a second phase to adequately assess the
finalists.&nbsp; In future challenges, depending on the
nature and scope, the challenge team may consider adjusting
the value of the overall prize pool and including prizes for
finalists and runners up.</p>
<p><strong>Area of Excellence #2: "Create an Implementation
Plan"</strong></p>
<p>The implementation plan for this challenge was developed
through ongoing consultation between the Division of
Foodborne, Waterborne, and Enteric Diseases (DFWED), the
Division of Healthcare Quality Promotion (DHQP), and the
Office of Advanced Molecular Detection (OAMD).&nbsp; The
challenge team developed the challenge question and
evaluation criteria, drafted all project documents and web
content and coordinated the technical review.</p>
<p>This collaboration on the design and implementation of the
challenge was important, because while these different
stakeholder groups all had shared interest in metagenomics
and culture-independent diagnostics, their specific
objectives and priorities differed.&nbsp; In drafting the
implementation plan for this challenge, the project team was
able to align objectives and evaluation criteria, resulting
in improvements to the design of the challenge and a higher
level of engagement from CDC stakeholders. In retrospect,
dedicated support personnel or contract support would have
helped this challenge run more smoothly, since the CDC core
challenge team was small and burdened with other
responsibilities.&nbsp; Even so, this was a successful
challenge, with a very successful outcome.</p>
<p><strong>Area of Excellence #3: "Manage
Solutions"</strong></p>
<p>Following the challenge competition, CDC subject matter
experts from the Division of Foodborne, Waterborne and
Environmental Diseases (DFWED) and the Office of Advanced
Molecular Detection (OAMD) worked with the winning team to
develop and refine several of the new platform features that
were developed as a result of the challenge, to improve
reference database curation and to assess the utility of the
software for the analysis of a variety of primary clinical
and environmental samples.</p>
<p>CDC and OneCodex are continuing to collaborate on the
development of software tools to analyze genomic sequencing
data from complex metagenomic samples.&nbsp; While its
target customer base is clinical diagnostics, the challenge
competition encouraged OneCodex to develop a number of
features that will be of direct benefit to public health
applications, and CDC is confident that the challenge
competition has secured lasting interest and commitment from
OneCodex's leadership and scientific team.</p>
<p>The most significant barrier to long-term engagement with
this solution is not scientific or technical, but rather
current limitations of federal cloud-computing policy.&nbsp;
The OneCodex platform, which runs as a Software as a Service
(SaaS) offering on top of Amazon Web Services, does not have
the necessary FedRAMP approvals for routine standalone use.&nbsp;
Consequently, CDC programs must continue to work with the
enterprise information security office to determine how to
authorize access to its software and other similar tools.&nbsp;
In the meantime, CDC has been collaborating with the company
and has conducted pilot tests on its platform using
carefully selected non-sensitive and de-identified or
simulated data.</p>
<h2>Challenge Type</h2>
<p><strong>Scientific</strong></p>
<p>This challenge was a success on several important levels.
First, it resulted in the identification of a software
solution that met an important and emerging public health
need and established an important precedent for engagement,
innovation and collaboration between public health and the
biotech startup community.&nbsp; Second, the competition
elicited significant interest around the public health
challenges posed by new non-culture-based clinical
diagnostic assays and has helped to foster academic research
around potential solutions.&nbsp; And third, it has helped
to establish challenge competitions and public-private
partnerships as key drivers for innovation at the
agency.</p>
<h2>Legal Authority</h2>
<p>America COMPETES Act</p>
<h2></h2>
<h2>Challenge Website</h2>
<p><a href="http://www.cdc.gov/amd/achievements/cidtchallenge">http://www.cdc.gov/amd/achievements/cidtchallenge</a>
</p>

</div>
</div>
</div>
</section>
<section class="container">
<div class="row">
<div class="blank_spacer col-sm-12 " style="height:30px;"></div>
</div>
</section>
<section class="container">
<div class="row">
<div class="blank_spacer col-sm-12 " style="height:30px;"></div>
</div>
</section>

</div>
</section>
</section>



</div>


</div>





<!-- CLOSE article -->
</article>

</div>


<!--// WordPress Hook //-->

<!--// CLOSE #page-wrap //-->
